src/main/java/uk/gov/justice/hmpps/prison/api/resource/ImageResource.java

```diff
@@ -47,7 +47,7 @@ public class ImageResource {
             @ApiResponse(responseCode = "404", description = "Requested resource not found."),
             @ApiResponse(responseCode = "500", description = "Unrecoverable error occurred whilst processing request.")})
     @Operation(summary = "Image data (as bytes).", description = "Requires role VIEW_PRISONER_DATA.")
-    @PreAuthorize("hasRole('ROLE_VIEW_PRISONER_DATA')")
+    @PreAuthorize("hasRole('VIEW_PRISONER_DATA')")
```

Comment:
Making it consistent - we normally omit ROLE_

Reply:
cool